Solutions/CohesitySecurity/Playbooks/Cohesity_Restore_From_Last_Snapshot/azuredeploy.json

```diff
@@ -0,0 +1,363 @@
+                                    "vmwareParameters": {
+                                        "powerOffAndRenameExistingVm": true,
+                                        "poweredOn": true,
+                                        "prefix": "Recover-",
+                                        "recoveryProcessType": "kCopyRecovery",
+                                        "suffix": "-VM"
+                                    }
+                                },
+                                "headers": {
+                                    "Content-Type": "application/json",
+                                    "apiKey": "@body('Get_secret')?['value']",
+                                    "clusterid": "@{body('Get_cid_from_blob_content')}"
+                                },
+                                "method": "POST",
+                                "uri": "https://helios.cohesity.com/irisservices/api/v1/public/restore/recover"
+                            }
+                        },
+                        "Initialize_Description": {
+                            "runAfter": {},
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [{
+                                    "name": "description",
+                                    "type": "string",
+                                    "value": "@triggerBody()?['object']?['properties']?['description']"
+                                }]
+                            }
+                        },
+                        "Initialize_HelioID": {
+                            "runAfter": {
+                                "Initialize_Description": [
+                                    "Succeeded"
+                                ]
+                            },
+                            "type": "InitializeVariable",
+                            "inputs": {
+                                "variables": [{
+                                    "name": "helioID",
+                                    "type": "string",
+                                    "value": "@{split(variables('description'), 'Helios ID: ')[1]}"
+                                }]
+                            }
+                        }
+                    },
+                    "outputs": {}
+                },
+                "parameters": {
+                    "$connections": {
+                        "value": {
+                            "azureblob": {
+                                "connectionId": "[resourceId('Microsoft.Web/connections', variables('AzureblobConnectionName'))]",
+                                "connectionName": "[variables('AzureblobConnectionName')]",
+                                "id": "[concat('/subscriptions/', subscription().subscriptionId, '/providers/Microsoft.Web/locations/', resourceGroup().location, '/managedApis/Azureblob')]"
+                            },
```

Comment:
Please check if ,this connection of azureblob be made of type "ManagedServiceIdentity" ? if possible

Reply:
I can't. I've got a similar error, like in the previous comment for Outlook.

```
{
    "status": "Failed",
    "error": {
        "code": "WorkflowManagedIdentityConfigurationInvalid",
        "message": "The workflow connection parameter 'azureblob' is not valid. The API connection 'azureblob' is not configured to support managed identity."
    }
}
```